larray/core/array.py

```diff
@@ -6306,8 +6306,10 @@ def combine_axes(self, axes=None, sep='_', wildcard=False):
 
         Parameters
         ----------
-        axes : tuple, list or AxisCollection of axes, optional
-            axes to combine. Defaults to all axes.
+        axes : tuple, list, AxisCollection of axes or list of combination of those or dict, optional
+            axes to combine. Tuple, list or AxisCollection will combine several axes into one. To chain several axes 
+            combinations, pass a list of tuple/list/AxisCollection of axes. To set name(s) of resulting axis(es), use a 
+            dictionary with entries {(axes to combine): 'new_axis_name'}. Defaults to all axes.
```

Comment:
maybe: s/use a dictionary with entries {(axes to combine): 'new_axis_name'}./use a {(axes, to, combine): 'new_axis_name'} dictionary./